### PR TITLE
Admin approve page

### PR DESCRIPTION
**Pull request by gakhromov**

### CODE REVIEW

File: front-end/src/api/admin.js [hunk 1]
```diff
@@ -31,3 +31,31 @@ export function saveUserPermissions(id, roles) {
...
+    url: BASE_API_URL + LMS_URLS.approve.await + `${id}`,
+    method: "post",
+  });
+}
+
+export function disapproveUser(id) {
+  return request({
+    url: BASE_API_URL + LMS_URLS.approve.decline + `${id}`,
+    method: "post",
+  });
```
Comment (TmLev) [inline front-end/src/api/admin.js:60]: The trailing slash is missing; Django will respond with 302.

File: front-end/src/constants/api.js [hunk 2]
```diff
@@ -50,6 +50,12 @@ export const LMS_URLS = {
     users: "lms/users/",
     permissions: "lms/users/permissions",
   },
+  approve: {
+    approve: "lms/activate-students",
```
Comment (TmLev) [inline front-end/src/constants/api.js:54]: Also, trailing slash.

Comment (gakhromov) [reply front-end/src/constants/api.js]: Added.

File: front-end/src/api/admin.js (hunk 1 above)
Comment (gakhromov) [reply front-end/src/api/admin.js]: Added.

**Review by TmLev — APPROVED** [review]
Great, thanks!

**Pull request merged**